docs/storage-backend/scylladb.md

```diff
@@ -27,3 +27,69 @@ ScyllaDB is fully compatible with Cassandra. To use it as the data storage layer
 - Spinning up a three-node Scylla Cluster and setting it as the data storage for the JanusGraph server
 - Performing some basic graph operations
 
+## Usage of Scylla optimized driver
+
+JanusGraph provides `scylla` storage.backend options in addition to generic `cql` option.  
+ScyllaDB can work using any of those storage options, but the dedicated `scylla` backend is better optimized 
+for ScyllaDB. Thus, it's recommended to use `scylla` backend option with ScyllaDB whenever possible.  
+`scylla` storage option is included in `janusgraph-scylla` library but this library isn't shipped via `janusgraph-all` 
+and this library is not part of the provided JanusGraph distributions.  
+To make `scylla` option available in JanusGraph it's necessary to replace `org.janusgraph:janusgraph-cql` 
+with `org.janusgraph:janusgraph-scylla` if JanusGraph is used in Embedded Mode.   
+
+In case `janusgraph-all` dependency is used instead of explicit JanusGraph dependencies then the replacement to
+`org.janusgraph:janusgraph-scylla` can be done like below.
+
+```xml tab='Maven'
+<dependencies>
+    <!-- Exclude `janusgraph-cql` from `janusgraph-all` -->
+    <dependency>
+        <groupId>org.janusgraph</groupId>
+        <artifactId>janusgraph-all</artifactId>
+        <version>1.0.0</version>
+        <exclusions>
+            <exclusion>
+                <groupId>org.janusgraph</groupId>
+                <artifactId>janusgraph-cql</artifactId>
+            </exclusion>
+        </exclusions>
+    </dependency>
+
+    <!-- Include `janusgraph-scylla` -->
+    <dependency>
+        <groupId>org.janusgraph</groupId>
+        <artifactId>janusgraph-scylla</artifactId>
+        <version>1.0.0</version>
+    </dependency>
+</dependencies>
+```
+
+```groovy tab='Gradle'
+/* Exclude `janusgraph-cql` from `janusgraph-all` */
+compile("org.janusgraph:janusgraph-all:1.0.0") {
+    exclude group: 'org.janusgraph', module: 'janusgraph-cql'
+}
+/* Include `janusgraph-scylla` */
+compile "org.janusgraph:janusgraph-scylla:1.0.0"
+```
+
+#### Using Scylla driver in JanusGraph Server provided via the standard distribution builds
+
+!!! note
+    The manual process described below is temporary and should be replaced by automated process after the issue
+    [janusgraph/janusgraph#3580](https://github.com/JanusGraph/janusgraph/issues/3580) is resolved.
```

Comment:
Would be really great if we could get this resolved until the 1.0.0 release.

Reply:
Agree. I don't really want users to handle manual copy/paste process. I think we should just change the distribution build process to have `lib`, `cqlLib`, `scyllaLib` folders as noted in #3580 
It should be fairly simple for users to switch between CQL or Scylla drivers by adding an optional environment variable. 
@farodin91 also [mentioned](https://github.com/JanusGraph/janusgraph/pull/3578#issuecomment-1428032096) using Java Shadowing approach which could potentially be an alternative version to resolve #3580 , but I haven't had a chance to investigate the approach with Java Shadowing.